corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/models.js

```diff
@@ -11,6 +11,17 @@ FormplayerFrontend.module("Users.Models", function(Models, FormplayerFrontend, B
             }.bind(this));
         },
 
+        isMobileWorker: function() {
+            return this.username.endsWith('commcarehq.org');
```

Comment:
i would, but unfortunately we don't have a great way of getting django variables into javascript files at the moment and i don't think it's worth the complexity of passing that arg around all the way to the model